airbyte-integrations/connectors/source-public-apis/acceptance-test-config.yml

```diff
@@ -10,17 +11,14 @@ acceptance_tests:
       - config_path: "secrets/config.json"
         status: "succeed"
       - config_path: "integration_tests/invalid_config.json"
-        status: "failed"
+        status: "succeed"
   discovery:
     tests:
       - config_path: "secrets/config.json"
   basic_read:
     tests:
       - config_path: "secrets/config.json"
         configured_catalog_path: "integration_tests/configured_catalog.json"
-        empty_streams: []
-  incremental:
-    bypass_reason: "This connector does not implement incremental sync"
```

Comment:
why was this removed?

Reply:
The connector doesn't support incremental.